packages/terra-framework-docs/src/terra-dev-site/doc/compact-interactive-list/Examples.2/FixedWidthColumns.1.doc.mdx

```diff
@@ -0,0 +1,15 @@
+import FixedWidthColumns from './FixedWidthColumns?dev-site-examples';
+
+# Fixed Width Columns
+
+### Description
+
+This example demonstrates a [Compact Interactive List](/components/cerner-terra-framework-docs/compact-interactive-list/about) which renders a collection of interactive data in a list format into a single tab stop. 
+
+* The list is arranged in 2 visual columns by the [_numberOfColumns_](/components/cerner-terra-framework-docs/compact-interactive-list/about#compact-interactive-list-props-table) prop set to `2`.
+* Each visual column consists of 3 semantic columns, such as Icon, Text Description, and an interactive Featured column.
+* All semantic columns have [_width_](/components/cerner-terra-framework-docs/compact-interactive-list/about#column) prop explicitly set, with the [_flexGrow_](/components/cerner-terra-framework-docs/compact-interactive-list/about#column) prop omitted. The columns will keep their width and not grow wider or shrink.
+* Horizontal scroll will show up if screen width is less than the width of the list.
+* The list is a one tab stop. The user can navigate between cells with the arrow keys. **Space key** press or mouse click on the Featured cell will toggle their state. **Esc key** press while on the list would unselect all featured items.
```

Comment:
Can we reword this slightly:
```suggestion
* The list is a one tab stop. The user can navigate between cells with the arrow keys. **Space key** press or mouse click on a Featured cell will toggle its state. **Esc key** press while on the list will unselect all featured items.
```

Reply:
Great suggestion, and much appreciated, but it looks like all wording and styling comment were outdated per this commit by @cldeitrich: https://github.com/cerner/terra-framework/pull/1950/commits/62c72c0098e6edb05ec923b22cf997650ac247bf